library/src/otx/data/entity/sample.py

```diff
@@ -99,6 +99,13 @@ class ClassificationSample(OTXSample):
     label: torch.Tensor = label_field(pl.Int32())
 
 
+class ClassificationMultiLabelSample(OTXSample):
+    """ClassificationMultiLabelSample is a base class for OTX multi label classification items."""
+
+    image: np.ndarray | tv_tensors.Image = image_field(dtype=pl.UInt8)
+    label: np.ndarray | torch.Tensor = label_field(pl.Int32(), multi_label=True)
```

Comment:
Should it be named `labels` instead of `label`? By the way, can you show an example how to instantiate a `ClassificationMultiLabelSample` with two labels? How does the `multi_label` parameter work?